test/fixtures/profiles/profile-with-gem-dependency/inspec.yml

```diff
@@ -0,0 +1,13 @@
+name: profile-with-gem-dependency
+title: InSpec Profile
+maintainer: The Authors
+copyright: The Authors
+copyright_email: you@example.com
+license: Apache-2.0
+summary: An InSpec Compliance Profile
+version: 0.1.0
+supports:
+  platform: os
+gem_dependencies:
+- name: money
+  version: "= 6.16.0"
```

Comment:
I think this should be loosened to be ">= 6.16.0" - pinning it "=" may break in the future.

Reply:
Updated to ">= 6.16.0"